### PR TITLE
FE-703 - Color and speed-up our test-ci script

### PR DESCRIPTION
**Pull request by nicklemmon**
[FE-703](https://jira.int.messagesystems.com/browse/FE-703)

### What Changed
- Added `--colors` flag to Jest command
- Updated Jest to latest version
- Updated Node to more recent version in the CI config
- The `--runInBand` flag produced no discernible speed improvements
- The `--maxWorkers` flag can't be used with our current Travis plan - with a paid plan we could potentially use it

### How To Test
- Check out the Travis output to see colors: https://travis-ci.org/SparkPost/2web2ui/builds/575330979?utm_source=github_status&utm_medium=notification
- Compare the test runtime between two Travis builds:
    * FE-703: https://travis-ci.org/SparkPost/2web2ui/builds/577855657
    * Master: https://travis-ci.org/SparkPost/2web2ui/builds/577567808

### To Do
- [ ] Incorporate feedback

### CODE REVIEW

**Comment by nicklemmon** [comment]
Waiting on #1156 before implementing

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "8"
+  - "10.8"
```
Comment (bkemper) [inline .travis.yml]: Should we bump "engines" and strictly enforce?

**Comment by nicklemmon** [comment]
@bkemper Ready to re-review - I updated `"engines"` to mirror our CI config. The `.npmrc` was already set with:

```
engine-strict=true
```

**Comment by bkemper** [comment]
@nicklemmon This is good to merge when you are ready.

**Pull request closed without merging**